### ISSUE TITLE
Slideshow nugget

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#56]
# Done criteria - MRM and BGS
- [x] Build and style nugget
- [ ] Enable a slide show to be created an published at a later date (@russ-manta can this be incorporated? NO. only scheduling of nodes can be implemented, not fields.

# Designs
- Design for [slideshow nugget](https://invis.io/4EDQ458R5#/273192591_Slideshow-Nugget)
- Also appears on [Resources](https://invis.io/4EDQ458R5#/272049140_Resources)
- [Example in Drupal](http://bgs-dev.mrmdev.co.uk/image-gallery)

**Comment by angievanhegan** [comment]
@cat-manta Please can you add a link when the design is done and has been agreed by @StephMSlack in #44 . Thanks

**Comment by angievanhegan** [comment]
@ican-icant do you like these?

**Comment by ican-icant** [comment]
I do indeed @angievanhegan I prefer the resources version with the tick boxes to the right of the search box.

**Comment by angievanhegan** [comment]
@ican-icant Are you happy with the arrows below the slideshow? https://projects.invisionapp.com/share/4EDQ458R5#/273192591_Slideshow-Nugget

**Comment by ican-icant** [comment]
Yes, they are nice @angievanhegan

**Comment by StephMSlack** [comment]
Hi @angievanhegan I'm really sorry, but I really don't like having the arrows and numbers below the slideshow. Some samples which are closer to what I was after are:
https://www.nationaltrust.org.uk/
https://www.rcplondon.ac.uk/

**Comment by angievanhegan** [comment]
@russ-manta @ican-icant would we be able to modify the slideshow as per steph's request so that it just has right and left arrows on the image?

**Comment by angievanhegan** [comment]
@StephMSlack Do you mean like the slideshow on the home page? http://bgs-dev.mrmdev.co.uk/

**Comment by StephMSlack** [comment]
Hi @angievanhegan this is definitely closer to what I was after. It depends if it is a lot of work to change the system. I'd still prefer the versions on the original links I sent through, the home page version is closer and I would prefer that to the original style. The integration work and building the other pages is a higher priority, but if this is a 1/2 day job to update, it would be good to make the change, but if it is 3 days worth of work, there are other things that need to be done first.

**Comment by angievanhegan** [comment]
@russ-manta the design for this slideshow has been updated (see the link at the top). Please can you estimate this please? Thanks

**Comment by angievanhegan** [comment]
@StephMSlack Russ confirmed this would be about half a days work. I'm assuming we need to add text for each image though? Especially if you're using them for the history of BGS as per Recia's comment? Thanks

**Comment by angievanhegan** [comment]
@cat-manta there needs to be a caption for the images in the slideshow. Can you add that to the design please? thanks.

**Comment by angievanhegan** [comment]
@StephMSlack Are you happy to sign off this design? Thanks

**Comment by StephMSlack** [comment]
yes the design is fine

**Comment by angievanhegan** [comment]
@cutlern This is ready to be built and styled

**Comment by meldera** [comment]
http://bgs-dev.mrmdev.co.uk/test

only scheduling of nodes can be implemented, not fields.

**Comment by angievanhegan** [comment]
@StephMSlack This is ready for you to review and sign off. Let me know if you need any help editing it.

**Comment by StephMSlack** [comment]
@angievanhegan This is huge! 

Is it possible to be able to choose a size? EG - this size would probably work really well for a series of posters from our conference.

But if we wanted a slideshow of event images with some text shaping around it, the images would be way too large. (I assume we could have it left or right aligned with text shaping around it?)

**Comment by angievanhegan** [comment]
@StephMSlack It's just a full column slideshow (with a right hand column), like the design. But I know what you mean! I'd assumed that if you have a slideshow it's because you're wanting to really show off the images and have people look at them. If the images are to add colour and interest to a page, I'd imagine that you'd be adding static images with the text flowing around them. Do you need us to reduce the height a bit, or is it worth talking through this requirement more?

**Comment by StephMSlack** [comment]
@angievanhegan will we be able to add the photo gallery nugget to an event page? We are looking at using the gallery nugget as post event content where we can add our event abstract posters.

With that in mind - is there anyway that we can add PDFs to the gallery? turning a pdf into a jpeg/png that is readable becomes a very large file, which becomes very slow to load on the page.

Also is there anything we can do to make portrait images appear a little better? many posters are portrait v's landscape

Is it possible for the title to be left aligned?
My sample with posters is on http://bgs-dev.mrmdev.co.uk/about/sandbox/stephs-test-page

**Comment by angievanhegan** [comment]
@StephMSlack I've created a new issue (#170 and captured all these requirements in it so that when we come to the events section we can make sure it can take a PDF, a portrait image (and we can tweak the text alignment while we're at it). I'm assuming I can sign this off for now though so I'm moving it to done.
